docs/developers/emerald-paratime/writing-dapps-on-emerald.md

```diff
@@ -34,10 +34,10 @@ account to Emerald. Consult the [How to transfer ROSE into Emerald ParaTime]
 [how-to-deposit-rose] chapter to learn more.
 
 
-[overview]: ../../oasis-network/overview.md
+[overview]: /general/oasis-network/
```

Comment:
do we care that this breaks links when viewing in github?

Reply:
Currently, `../../oasis-network` would point outside of the *developers* component and docusaurus won't let you compile this. We could remove the separate @docusaurus/plugin-content-docs per component and take docs/ folder as the root for everything. I would need to think about it, if that breaks anything else.

FYI referencing markdown files policy is defined here: https://github.com/oasisprotocol/docs/blob/matevz/feature/restructure-docs/README.md#referencing-documents-between-the-components

Reply:
Ok, I "unified" all content plugins now so the cross-component relative links to .md files should work.